build/pipelines/templates/build-console-steps.yml

```diff
@@ -70,7 +70,7 @@ steps:
   displayName: 'Check MSIX for common regressions'

```

Comment:
meh if we're going to disable 70% of the rules in this file, I would rather we just put the steps directly into build-console-fuzzing itself.

Reply:
I'm mixed on that. I hear what you're saying, but this mess might make it clearer when we give the YAML pipelines some all-up-love of how to more clearly detangle them (by showing the most tangled parts to the future reviser... who is probably me.) But also it does feel a little silly.

Reply:
I'm not sure I subscribe to the "we have to let it get worse before we make it better" line of thinking. We're adding more tangle and more knots to our one uber-pipeline instead of making a smaller reusable component, which the disentangler will need to disentangle once they start. I get what you mean, though. I dunno.

Reply:
I'm happy to clean up the YAML pipelines, actually. For now, I'll throw them into build-console-fuzzing.yml since it's only like 8 steps. I'll clean up our pipelines more in a follow-up.